src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.figures.Triangle;
+import java.util.Random;
+
+class FigureSupplier {
+    Figure getRandomFigure() {
```

Comment:
let's add access modifiers to your classes and methods (public, protected or private), do it everywhere